medicines/api/src/document.rs

```diff
@@ -1,16 +1,53 @@
-use crate::{pagination, pagination::PageInfo, product::Product};
+use crate::{pagination, pagination::PageInfo};
+use juniper::GraphQLObject;
 use search_client::{models::IndexResult, Search};
 
-#[derive(juniper::GraphQLObject)]
+#[derive(GraphQLObject, Eq, Ord, PartialEq, PartialOrd)]
 #[graphql(description = "A document")]
 pub struct Document {
-    name: String,
-    // Yes, more fields will be added here.
+    product_name: Option<String>,
+    active_substances: Option<Vec<String>>,
+    title: Option<String>,
+    highlights: Option<Vec<String>>,
+    created: Option<String>,
+    doc_type: Option<String>,
+    file_bytes: Option<i32>,
```

Comment:
I guess it'll be obvious when using the API, but this might be better named `file_size` or `file_size_in_bytes`.

Reply:
Perhaps we should re-visit these field names once we've hooked up actual data to the API.